static/js/add-shows/new-show.js

```diff
@@ -5,9 +5,9 @@ MEDUSA.addShows.newShow = function() {
         var showName;
         var sepChar;
         // if they've picked a radio button then use that
-        if ($('input:radio[name=whichSeries]:checked').length) {
+        if ($('input:radio[name=whichSeries]:checked').length !== 0) {
```

Comment:
@OmgImAlexis why use strict comparison (not equal value or not equal type) when `.length` will always return INT ? can't we use just `!=` ?

is there any issue using existing code?